packages/firestore/test/lite/integration.test.ts

```diff
@@ -2227,101 +2222,77 @@ describe('countQuery()', () => {
-      const countQuery1 = countQuery(query1);
-      const countQuery2 = countQuery(query2);
-      expect(aggregateQueryEqual(countQuery1, countQuery2)).to.be.true;
-    });
+  it('aggregateFieldEqual on same aggregate fields', () => {
+    const AggregateField1 = count();
+    const AggregateField2 = count();
+    expect(aggregateFieldEqual(AggregateField1, AggregateField2)).to.be.true;
   });
 
-  it('aggregateQueryEqual on different queries', () => {
-    const testDocs = [
-      { author: 'authorA', title: 'titleA' },
-      { author: 'authorA', title: 'titleB' },
-      { author: 'authorB', title: 'titleC' }
-    ];
-    return withTestCollectionAndInitialData(testDocs, async coll => {
-      const query1 = query(coll, where('author', '==', 'authorA'));
-      const query2 = query(coll, where('author', '==', 'authorB'));
-      const countQuery1 = countQuery(query1);
-      const countQuery2 = countQuery(query2);
-      expect(aggregateQueryEqual(countQuery1, countQuery2)).to.be.false;
-    });
+  it('aggregateFieldEqual on different aggregate fields', () => {
+    const AggregateField1 = count();
+    const AggregateField2 = new AggregateField('test');
+    expect(aggregateFieldEqual(AggregateField1, AggregateField2)).to.be.false;
   });
 
-  it('aggregateQuerySnapshotEqual on same queries', () => {
+  it('aggregateSnapshotEqual on same queries', () => {
     const testDocs = [
       { author: 'authorA', title: 'titleA' },
       { author: 'authorA', title: 'titleB' },
       { author: 'authorB', title: 'titleC' }
     ];
-    return withTestCollectionAndInitialData(testDocs, async coll => {
-      const query1 = query(coll, where('author', '==', 'authorA'));
-      const query2 = query(coll, where('author', '==', 'authorA'));
-      const countQuery1A = countQuery(query1);
-      const countQuery1B = countQuery(query1);
-      const countQuery2 = countQuery(query2);
-      const snapshot1A = await getAggregate(countQuery1A);
-      const snapshot1B = await getAggregate(countQuery1B);
-      const snapshot2 = await getAggregate(countQuery2);
-      expect(aggregateQuerySnapshotEqual(snapshot1A, snapshot1B)).to.be.true;
-      expect(aggregateQuerySnapshotEqual(snapshot1A, snapshot2)).to.be.true;
+    return withTestCollectionAndInitialData(testDocs, async collection => {
+      const query1 = query(collection, where('author', '==', 'authorA'));
+      const query2 = query(collection, where('author', '==', 'authorA'));
+      const snapshot1A = await getCountFromServer(query1);
+      const snapshot1B = await getCountFromServer(query1);
+      const snapshot2 = await getCountFromServer(query2);
+      expect(aggregateSnapshotEqual(snapshot1A, snapshot1B)).to.be.true;
+      expect(aggregateSnapshotEqual(snapshot1A, snapshot2)).to.be.true;
     });
   });
 
-  it('aggregateQuerySnapshotEqual on different queries', () => {
+  it('aggregateSnapshotEqual on different queries', () => {
```

Comment:
I just thought of this... can you add a test case for `aggregateSnapshotEqual()` where the queries are different but the count is the same? This only needs to be added to the lite sdk's test suite.

Reply:
changed the testDocs to get the same count result with different query.